esmvaltool/diag_scripts/primavera/energy_budget.py

```diff
@@ -0,0 +1,249 @@
+import os
+import logging
+
+import matplotlib
+matplotlib.use('Agg')  # noqa
```

Comment:
```suggestion
```
This is set using an environmental variable by esmvaltool when running the script, so there should be no need to do it in the code here